### PR TITLE
Revert "Restored etc/install-tools.sh"

### PR DESCRIPTION
**Pull request by fingolfin**
This reverts commit 20169c70cfc503f2d38a31402b8fa64d3a65787d.

Do not merge right now, please see issue #1118 first.

### CODE REVIEW

**Comment by fingolfin** [comment]
Note that `doc/ref/helpintf.xml` mentions `install-tools.sh`, and should be adjusted.

**Comment by alex-konovalov** [comment]
This is yet an incomplete PR, so we need to carefully identify what has to be done in this repository and in gap-distribution. Is the plan then to:

1) in this repository update both `etc/README.tools` and `doc/ref/helpintf.xml` to not to mention tools.tag.gz

2) in gap-distribution: ensure that the main archive contains `etc/convert.pl`, `doc/gapmacrodoc.tex`, `doc/gapmacro.tex`, `doc/manualindex` and also `doc/gapmacrodoc.pdf` which would be produced from `doc/gapmacrodoc.tex` during the release wrapping. Update release wrapping scripts to not to make tools archive.

3) check also what Example package says about tools archive.

I think it's worth efforts, it will make the release wrapping more streamlined. Of course the fact that we include `etc/convert.pl`, `doc/gapmacrodoc.tex`, `doc/gapmacro.tex`, `doc/manualindex` does not mean that we support them!

**Comment by fingolfin** [comment]
I added another commit to this PR here which edits `doc/ref/helpintf.xml`. Note that `etc/README.tools`  was already changed to not mention `install-tools.sh` back in PR  #1064.

File: doc/ref/helpintf.xml [hunk 1]
```diff
@@ -90,25 +90,20 @@ any more. The book can be re-installed using <Ref Func="HELP_ADD_BOOK" />. <P/>
 <Section Label="The manual.six File">
 <Heading>The manual.six File</Heading>
 
-If a <F>manual.six</F> file for a help book is not in the format of the
-<F>gapmacro.tex</F> macros, explained in the document 
-<Q>The gapmacro.tex Manual Format</Q> (see the file 
-<F>gap4r5/doc/gapmacrodoc.pdf</F> from the tools archive
```
Comment (alex-konovalov) [inline doc/ref/helpintf.xml]: So, do you prefer to throw away mentioning of `doc/gapmacrodoc.pdf` completely? I thought we build it and include in the GAP source distribution.

File: doc/ref/helpintf.xml [hunk 2]
```diff
@@ -90,25 +90,20 @@ any more. The book can be re-installed using <Ref Func="HELP_ADD_BOOK" />. <P/>
...
-which is the <C>default</C> format.
+where <A>myownformat</A> is an identifying string for this format. The reading
+of the (remainder of the) file is then delegated to the function
+<C>HELP_BOOK_HANDLER.<A>myownformat</A>.ReadSix</C> which must exist. Thus there
+are no further regulations for the format of the <F>manual.six</F> file, other
+that what you yourself impose.
+<P/>
+If such a line is missing then it is assumed that the <F>manual.six</F> file
+complies with the <F>gapmacro.tex</F> documentation format which is the
+<C>default</C> format, and reading the file is delegated to
```
Comment (alex-konovalov) [inline doc/ref/helpintf.xml]: `the <F>gapmacro.tex</F> documentation  format which is the <C>default</C> format.` - it seems that this was outdated  statement in the old version, copied to the new one.

**Comment by alex-konovalov** [comment]
@fingolfin wrote: 

> Note that etc/README.tools was already changed to not mention install-tools.sh back in PR #1064.

True, but it still mentions `tools.tar.gz` archive a couple of times.

File: doc/ref/helpintf.xml (hunk 1 above)
Comment (fingolfin) [inline doc/ref/helpintf.xml]: Test, I want to throw that away (and also the reference to the dir name `gap4r5`). After all, this file is only of interest for authors of legacy packages, who already should know about it, or be able to find it, or be able to ask us where it is. But I do not want new GAP developers to accidentally stumbler over it and perhaps even get the wrong idea that using it is encouraged.

File: doc/ref/helpintf.xml (hunk 2 above)
Comment (fingolfin) [inline doc/ref/helpintf.xml]: Not sure what you mean here. This is factual: the string "default" is the type id for documentation using `gapmacro.tex`. Note that the term "default" here is simply an identifier string, it does not mean that this format is what people should us "by default".

**Comment by fingolfin** [comment]
Closed in favor of #1305.

**Pull request closed without merging**